backends/qualcomm/tests/models.py

```diff
@@ -416,6 +416,17 @@ def forward(self, x):
         return torch.sum(self.first(x), dim=(2, 3), keepdim=False)
 
 
+class Conv2dTopK(torch.nn.Module):
```

Comment:
I assume it's just to test this kind of common pattern, and they won't be ussed?

Reply:
Yes. This is just for testing purpose to check layout transform is working properly.